### PR TITLE
Added county

### PR DESCRIPTION
**Pull request by TheMrM**
Implemented county section for the fallowing companies Altom | Avangardesoftware | Cartofiserie | PPT | Tremend

### CODE REVIEW

File: sites/cartofisserie.py [hunk 1]
```diff
@@ -17,16 +18,20 @@
     job_url = job.find('h2', class_='JCContentMiddle__Title').find('a')['href']
     job_url = 'https://www.ejobs.ro' + job_url
     job_location = job.find('span', class_='JCContentMiddle__Info').text.replace('\u0219', 's').split('si alte')[0].split(',')
+    cities = [city.strip() for city in job_location]
```
Comment (lalalaurentiu) [inline sites/cartofisserie.py:21]: Lipsesc orașe .
Screenshot 2023-11-15 at 15 12 33
Screenshot 2023-11-15 at 15 12 45

File: sites/tremend.py [hunk 2]
```diff
@@ -16,36 +17,27 @@
...
-                city = city,
-                country = 'Romania',
-                company = company
-            )
+    location_text = job.find('p', id='location-word').text.strip().replace('Location', '').strip()
+    remote = location_text == 'Remote'
+    city = None
+    county = None
+
+    if not remote:
```
Comment (lalalaurentiu) [inline sites/tremend.py:25]: joburile nu au orașe, județe sau câmpul remote.
Screenshot 2023-11-15 at 15 22 43
Screenshot 2023-11-15 at 15 24 20

File: sites/ppt.py [hunk 3]
```diff
@@ -11,22 +12,34 @@
...
 
 for job in job_elements:
     job_title = job.find('h2', class_='JCContentMiddle__Title').text.strip()
     job_url = job.find('h2', class_='JCContentMiddle__Title').find('a')['href']
     job_url = 'https://www.ejobs.ro' + job_url
     job_location = job.find('span', class_='JCContentMiddle__Info').text.replace('\u0219', 's').split('si alte')[0].split(',')
+    clean_job_location = [city.strip() for city in job_location]
+
+    counties = []
+    for city in clean_job_location:
```
Comment (lalalaurentiu) [inline sites/ppt.py:25]: Lipsesc Orașe.
Screenshot 2023-11-15 at 15 17 33

**Review by lalalaurentiu — CHANGES_REQUESTED** [review]

**Comment by TheMrM** [comment]
Cartofiserie scraper results
image
Tremend scraper results
image
PPT scraper results
image

File: sites/tremend.py [hunk 4]
```diff
@@ -1,46 +1,48 @@
...
-for job in job_elements.find_all('div', class_='career-wrapper'):
-    job_title = job.find('h3').text
-    job_link = url + job.find('a')['href']
+for job in job_elements:
+    job_title = job.find('h3').text.strip()
+    job_link = 'https://tremend.com/careers/' + job.find('a')['href'].strip('/')
     location_text = job.find('p', id='location-word').text.strip().replace('Location', '').strip()
-    remote = location_text == 'Remote'
+
+    remote = 'Remote' in location_text
```
Comment (lalalaurentiu) [inline sites/tremend.py:20]: campus remote trebuie sa conțină strungul "Remote" nu valoarea bool true.
Screenshot 2023-11-16 at 15 39 38

**Review by lalalaurentiu — CHANGES_REQUESTED** [review]

**Review by lalalaurentiu — APPROVED** [review]

**Pull request merged**